lib/libfrr.h

```diff
@@ -174,4 +174,36 @@ extern bool debug_memstats_at_exit;
 }
 #endif
 
+static inline time_t frr_get_realtime(struct timeval *tvo)
```

Comment:
#include "monotime.h"

monotime(tvo);

Please use the already built funtion.

Reply:
monotime() cannot be used to get real displayable time. It is monotonic time and is only useful to calculate time differences/intervals. In this case we need to report the actual time the cpu starvation/hog was seen. Hence the separate need for frr_get_realtime(). Hope it clarifies.

Reply:
Changed latest code to use monotime_to_real(). But still need frr_realtime_to_string() for printing realtime with microseconds granularity. Following frrtime_to_interval() example in monotime.h moved frr_realtime_to_string() to monotime.h.